cypress/integration/plugins/security-dashboards-plugin/multitenancy_shared_links.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { STACK_MANAGEMENT_PATH } from '../../../utils/dashboards/constants';
+
+import { switchTenantTo } from './switch_tenant';
+import 'cypress-real-events/support';
+
+Cypress.automation('remote:debugger:protocol', {
```

Comment:
Can't we in the cypress.json add:
```
    "browserPermissions": {
      "clipboard": "allow"
    }
```

That way other tests can just assume it can use the clipboard.

Reply:
+1